src/wallet/walletdb.cpp

```diff
@@ -465,6 +477,39 @@ ReadKeyValue(CWallet* pwallet, CDataStream& ssKey, CDataStream& ssValue,
 
             pwallet->LoadKeyPool(nIndex, keypool);
         }
+        else if (strType == "pending_firstupdate")
+        {
+            std::string strName, strJsonData;
+            ssKey >> strName;
+            ssValue >> strJsonData;
+            UniValue jsonData, v1, v2, v3;
```

Comment:
Please use more descriptive names than `v1`, `v2` and `v3`.  Or since these seem to be just used as a temporary value before converting to string, just have a single variable that gets reused?